### ISSUE TITLE
Misc description for Acheron Eidolons

### ISSUE THREAD

**Issue by ShalltearFR** [Mar-7th/StarRailRes#15]
hello, I just wanted to note that a description is missing for Acheron (ID: 130801) in french character_ranks.json file.
This concerns the index_min and index_new folder.

Thanks in advance for the little addition.

![image](https://github.com/Mar-7th/StarRailRes/assets/20574430/848f25cf-f4ac-4e50-a1c6-3c71b0011f16)

**Comment by mobyw** [comment]
Hi, the correct description is not updated in the upstream StarRailData, in file:

https://github.com/Dimbreath/StarRailData/blob/master/TextMap/TextMapFR.json

Current:

```json
{
    "-1828742606": "Augmente de <unbreak>#1[i]%</unbreak> les chances de coup critique lorsque le personnage inflige des DGT aux ennemis qui subissent des malus. Lorsque Rêve flétri atteint la limite, chaque point de Rêve flétri au-dessus de la limite accorde un cumul de Chemin de la réalisation, cumulable jusqu'à <unbreak>#2[i]</unbreak> fois."
}
```

Correct:

```json
{
    "-1828742606": "Augmente les chances de coup critique de <color=#f29e38ff><unbreak>#1[i]%</unbreak></color> lorsque le personnage inflige des DGT aux ennemis qui subissent des malus."
}
```

You can open an issue in the upstream repository, or just wait for the upstream to be updated when the next version is released.

**Comment by ShalltearFR** [comment]
Fixed in 2.2 update 👌
